id_minter/src/main/scala/uk/ac/wellcome/platform/idminter/database/IdentifiersDao.scala

```diff
@@ -16,22 +16,22 @@ class IdentifiersDao @Inject()(db: DB, identifiers: IdentifiersTable)
 
   implicit val session = AutoSession(db.settingsProvider)
 
-  def findSourceIdInDb(miroId: String): Future[Option[Identifier]] =
+  def lookupMiroID(miroID: String): Future[Option[Identifier]] =
```

Comment:
My current thought (with half an eye to ontologyType as I wrote this code) is that, at least for now, you stick an extra parameter on this method:

```scala
def lookupMiroID(miroID: String, ontologyType: String)
```

Further down, doing this in a way that doesn’t require us to hard-code each lookup as a method we might want would be nice, but I think hand-rolling these is still simpler (for now).